### DISCUSSION TITLE
Support Chia 1.6.2

### DISCUSSION THREAD

**Discussion by Blackcode** [dkackman/chia-dotnet#67] [Ideas]
Is this version supporting last version of Chia? I can't see any updates in 5 months.

Thank you,
D.E.

**Comment by dkackman** [comment]
I haven't updated it since 1.4.0. I have started using node and js for my projects and there didn't seem to be much demand for a /net client. Happy to revive things but would need help with testing and validation.

**Reply by Blackcode** [reply]
Hello dkackman,

I am using this library on chia and different chia forks for over a year now with no problems since is the most stable one. One example where this library is used is www.hddcoin.bet on HDD Blockchain.

I also wanna assure you that even it may not seem, such a stable .net library and c# generally is needed in the Chia ecosystem.

I would appreciate if you could maintain this library to the latest version of Chia and if you need I can help you testing.

Just let me know.

Greetings,
D.E.

**Reply by dkackman** [reply]
Cool. I will dust it off within the next few days and see what updates are needed.

**Comment by dkackman** [comment]
I've created a [feature](https://github.com/dkackman/chia-dotnet/issues/68) and [branch](https://github.com/dkackman/chia-dotnet/tree/1.6.2) for 1.6.2 if you want to follow along.

If there is a particular api you're looking for first please let me know here.

**Reply by Blackcode** [reply]
Ok, thanks as soon as I get home I follow.
In order of features I am interested of the CAT part but do in any order you like. I will try to help you testing.

Thanks again,
D.E.

**Comment by dkackman** [comment]
Hey @Blackcode - i don't know how much time I'm going to have to devote to this ongoing. I'd be happy to add you as a collaborator if you'd like to help maintain this library and keep it up to date.

**Reply by Blackcode** [reply]
@dkackman, thanks, I will try to update it even if lately had a lot of problems to sort...
Can you also tell me what is left for the 1.6.2 update?

**Reply by dkackman** [reply]
OH hey - This kinda fell dormant but is being picked back up. List of things needed to get up to date with 1.8.2 is here: 
https://github.com/dkackman/chia-dotnet/issues/80

@KevinOnFrontEnd is adding some things and has ideas on imp[roving the unit test setup.

**Reply by Blackcode** [reply]
@dkackman, great news, i will try to keep this up as sson as possible, had some private issues to solve

**Reply by dkackman** [reply]
support for 2.0.0 is done. unit test coverage is still not 100% but have made significant progress
https://www.nuget.org/packages/chia-dotnet/2.0.1

**Comment by dkackman** [comment]
main is now up to date with chia 1.8.2. Much testing left to do
